Makefile

```diff
@@ -86,10 +86,12 @@ ifneq ($(DEBUG_LEVEL), 2)
 OPT += -O2 -fno-omit-frame-pointer
 ifeq (,$(findstring ppc64,$(MACHINE))) # ppc64[le] doesn't support -momit-leaf-frame-pointer
 ifneq ($(MACHINE),armv7l)
+ifneq ($(MACHINE),s390x)
```

Comment:
this should be fixed with https://github.com/facebook/rocksdb/pull/1019. can you rebase?